mvvmcDevSprint/Controllers/DanielSeintenfus/Login/DSLoginViewController.swift

```diff
@@ -2,6 +2,8 @@ import UIKit
 
 class DSLoginViewController: UIViewController {
     
+    private var viewModel: DSLoginViewModel!
```

Comment:
`private let viewModel: DSLoginViewModel`

Reply:
O problema é que eu só posso atribuir a instância da viewModel à outra classe depois que ela for inicializada.
Como eu tenho esse trecho `coordinator.viewController = self`, porque ainda não apliquei injeção de dependência, eu só posso fazer isso depois de chamar o init da super classe

<img width="941" alt="Captura de Tela 2022-10-06 às 11 59 48" src="https://user-images.githubusercontent.com/20246441/194347637-e1cfaf5a-3589-48a2-99b1-8358d9224070.png">

Reply:
Ou tô viajando? haha

Reply:
saquei, então deixa do seu jeito mesmo